frontend/src/ches/MergeForm.jsx

```diff
@@ -711,6 +732,8 @@ class MergeForm extends Component {
 
     const senderPlaceholder = this.state.hasSenderEditor ? 'you@example.com' : this.state.config.sender;
 
+    const previewBodyDisplay = (this.state.apiValidationErrors && this.state.apiValidationErrors.length > 0) || (this.state.error && this.state.error.length >0) ? {display: 'none'} : {};
```

Comment:
Can multiline longer ternary operators for readability.

Reply:
will do on the next PR.